scripts/convert_data_format.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+"""Convert a recorded pickle file to the new format."""
```

Comment:
to the "new format" ? (what is new format ?)

Reply:
I extended to docstring.  Is it clear now?